modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -0,0 +1,171 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, mergeTests, test} from '@playwright/test';
+
+import {loginTest} from './loginTest';
+
+export type FeatureFlagsOptions = {
+	[key: string]: boolean;
+};
+
+export type FeatureFlags = FeatureFlag[];
+
+export interface FeatureFlag {
+	readonly enabled: boolean;
+	readonly key: string;
+}
+
+function featureFlagsTest(options: FeatureFlagsOptions) {
+	const fixtureImpl = test.extend<{
+		featureFlags: FeatureFlags;
+	}>({
+		featureFlags: [
+			async ({page}, use) => {
+
+				// Save original state of FFs
+
+				const originalFeatureFlags: FeatureFlagResult[] = [];
+
+				for (const key of Object.keys(options)) {
+					const result = await isEnabled(key, page);
+
+					originalFeatureFlags.push(result.featureFlag);
+					originalFeatureFlags.push(...result.dependentFeatureFlags);
+				}
+
+				try {
+
+					// Activate  FFs
+
+					for (const [key, enabled] of Object.entries(options)) {
+						await setEnabled(enabled, key, page);
+					}
```

Comment:
This is disabling FFs as well as activating, right? Comment may be putting me off.

Reply:
True. I created a first version that only enables but then people asked me to add disabling too and I didn't update the comment.